packages/keyvAdapter/src/__tests__/KeyvAdapter.test.ts

```diff
@@ -70,7 +70,15 @@ describe("KeyvAdapter", () => {
       expect(storeSetCalled).toBe(true);
     });
 
-    it("get", async () => {
+    it("get (batching enabled - default)", async () => {
```

Comment:
test that you can run two in parallel and that it works? (bonus points for testing that it actually calls getMany)